telcell/utils/transform.py

```diff
@@ -179,21 +179,11 @@ def select_colocated_pairs(tracks: List[Track],
     :param max_delay: the maximum amount of delay that is allowed.
```

Comment:
De naamgeving van deze functie is ambigu: zijn het track pairs of measurement pairs? Ook het werkwoord `select` is misleidend. Suggestie: `get_colocated_measurement_pairs`, of beter nog: `get_colocated_switches`
Geen probleem van deze PR maar wel van de moeder-PR. Wil je dat in deze branch meenemen?

Reply:
ik pas het aan!